### PR TITLE
fix OrderBundle depends on Core component #10319

### LINKED ISSUE #10319: OrderBundle depends on Core component
**Sylius version affected**: 1.4.1

**Description**  

`
Type error: Return value of Sylius\Bundle\OrderBundle\Controller\OrderController::getOrderRepository() must be an instance of Sylius\Component\Core\Repository\OrderRepositoryInterface, instance of Sylius\Bundle\OrderBundle\Doctrine\ORM\OrderRepository returned
`

**Steps to reproduce**  

use OrderBundle in standalone and overriding OrderRepository

### PR DESCRIPTION
**Pull request by embaya**
| Q               | A
| --------------- | -----
| Branch?         | 1.4
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #10319 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/OrderBundle/Controller/OrderController.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 use FOS\RestBundle\View\View;
 use Sylius\Bundle\ResourceBundle\Controller\RequestConfiguration;
 use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
-use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\Component\Order\Repository\OrderRepositoryInterface;
```
Comment (pamil) [inline src/Sylius/Bundle/OrderBundle/Controller/OrderController.php]: This makes the call in `summaryAction` incorrect - `$cart = $this->getOrderRepository()->findCartForSummary($cart->getId());`.

The solution which might work:
 * replace `findCartForSummary` with `findCartById` here
 * copy the existing method implementation to the `OrderController` in `CoreBundle`

Comment (embaya) [reply src/Sylius/Bundle/OrderBundle/Controller/OrderController.php]: I made the modifications you recommended to me

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "fix-#10319"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by embaya** [comment]
Thank you for your feedback, I don't have any commits to add. I hope these changes will solve the problem.

File: src/Sylius/Bundle/OrderBundle/Controller/OrderController.php (hunk 1 above)
Comment (hmonglee) [reply src/Sylius/Bundle/OrderBundle/Controller/OrderController.php]: @pamil is it ok for you ? thx

**Comment by pamil** [comment]
Thanks, Majed! :1st_place_medal:

**Pull request merged**